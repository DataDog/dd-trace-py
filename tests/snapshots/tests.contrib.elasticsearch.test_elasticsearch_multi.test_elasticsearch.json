[[
  {
    "name": "elasticsearch.query",
    "service": "elasticsearch",
    "resource": "PUT /ddtrace_index",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "elasticsearch",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "component": "elasticsearch",
      "elasticsearch.method": "PUT",
      "elasticsearch.params": "ignore=400",
      "elasticsearch.url": "/ddtrace_index",
      "http.status_code": "200",
      "language": "python",
      "out.host": "localhost",
      "runtime-id": "48d5633e5196471c87a429e6e232c9da",
      "span.kind": "client"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 1303
    },
    "duration": 108522700,
    "start": 1632159155469378300
  }],
[
  {
    "name": "elasticsearch.query",
    "service": "elasticsearch",
    "resource": "DELETE /ddtrace_index",
    "trace_id": 1,
    "span_id": 1,
    "parent_id": 0,
    "type": "elasticsearch",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "component": "elasticsearch",
      "elasticsearch.method": "DELETE",
      "elasticsearch.params": "ignore=%5B400%2C+404%5D",
      "elasticsearch.url": "/ddtrace_index",
      "http.status_code": "200",
      "language": "python",
      "out.host": "localhost",
      "runtime-id": "48d5633e5196471c87a429e6e232c9da",
      "span.kind": "client"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 1303
    },
    "duration": 57610400,
    "start": 1632159155578269400
  }]]
