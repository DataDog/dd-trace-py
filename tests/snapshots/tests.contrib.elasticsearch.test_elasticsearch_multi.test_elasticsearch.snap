[[{"name" "elasticsearch.query"
   "service" "elasticsearch"
   "resource" "PUT /ddtrace_index"
   "type" "elasticsearch"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1615404339005889000
   "duration" 94071000
   "meta" {"runtime-id" "f64590d3c3cf4b578d03f6de8cdf1b5c"
           "elasticsearch.method" "PUT"
           "elasticsearch.url" "/ddtrace_index"
           "elasticsearch.params" "ignore=400"
           "http.status_code" "200"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 51885
              "_dd.tracer_kr" 1.0}}]
 [{"name" "elasticsearch.query"
   "service" "elasticsearch"
   "resource" "DELETE /ddtrace_index"
   "type" "elasticsearch"
   "error" 0
   "span_id" 0
   "trace_id" 1
   "parent_id" nil
   "start" 1615404339100245000
   "duration" 46499000
   "meta" {"runtime-id" "f64590d3c3cf4b578d03f6de8cdf1b5c"
           "elasticsearch.method" "DELETE"
           "elasticsearch.url" "/ddtrace_index"
           "elasticsearch.params" "ignore=%5B400%2C+404%5D"
           "http.status_code" "200"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 51885
              "_dd.tracer_kr" 1.0}}]]
